platform_versioned_docs/version-24.3/studios/managing.mdx

```diff
@@ -49,84 +49,84 @@ To add a new data studio, complete the following steps:
+You can use any existing Studio as the foundation for adding a new Studio. This functionality creates a clone of the Studio, including its checkpoint history, preserving any modifications made to the original Studio. When you create a session in this way, future changes are isolated from the original session.
 
 When adding a new session from an existing session or checkpoint, the following fields cannot be changed:
 
-- **Data Studio Template**
-- **Original data studio session and checkpoint**
+- **Studio Template**
+- **Original Studio session and checkpoint**
 - **Compute environment**
 - **Installed Conda packages**
 
-To add a new session from an existing **stopped** session, complete the steps described in [Add a data studio](#add-a-data-studio).
+To add a new session from an existing **stopped** session, complete the steps described in [Add a Studio](#add-a-studio).
 
-Additionally, you can add a new session from any existing data studio checkpoint except the currently running checkpoint. From the data studio detail page, select the **Checkpoints** tab and in the **Actions** column, select **Start as new data studio**. This is useful for interactive analysis experimentation without impacting the state of the original data studio.
+Additionally, you can add a new session from any existing Studio checkpoint except the currently running checkpoint. From the detail page, select the **Checkpoints** tab and in the **Actions** column, select **Start as new Studio**. This is useful for interactive analysis experimentation without impacting the state of the original Studio.
 
-## Connect to a data studio
+## Connect to a Studio
 
 This functionality is available to all user roles excluding the **View** role.
 
-To connect to a running session, select the three dots next to the status message and choose **Connect**. A new browser tab will open containing the data studio session.
+To connect to a running session, select the three dots next to the status message and choose **Connect**.
 
 :::warning
 An active connection to a session will not prevent administrative actions that might disrupt that connection. For example, a session can be stopped by another workspace user while you are active in the session, the underlying credentials can be changed, or the compute environment can be deleted. These are independent actions and the user in the session won't be alerted to any changes - the only alert will be a server connection error in the active session browser tab.
 :::
 
-Once connected, the data studio session will display the status of **running** in the list, and any connected user's avatar will be displayed under the status in both the list of data studios and in each data studio session detail page.
+Once connected, the session will display the status of **running** in the list, and any connected user's avatar will be displayed under the status in both the list of Studios and in each Studio's detail page.
```

Comment:
@robnewman, is it a list of Studios or Studio sessions, and is it each "Studio's detail page" or "each Studio session's detail page"?

Reply:
Based on the most recent discussion, it's a `list of Studios` and it's `each Studio's detail page`.

Reference: https://seqera.slack.com/archives/C040MFU06AE/p1740407438484439?thread_ts=1739987472.109469&cid=C040MFU06AE